app/renderer/actions/Session.js

```diff
@@ -580,13 +585,15 @@ export function saveSession (caps, params) {
       for (let session of savedSessions) {
         if (session.uuid === uuid) {
           session.caps = caps;
+          session.server = server;
+          session.serverType = serverType;
```

Comment:
would that still work for older saves?

Reply:
just adding new fields on the object, so it should work. i'll pull it down and check on my current saves to be sure before merging.